app/views/macros/search-filter-tags.njk

```diff
@@ -1,6 +1,6 @@
 
 {% macro searchFilterTags(model) %}
-  <div class='govuk-body mpdp-display-grid'>
+  <div class='pdp-display-grid'>
```

Comment:
I believe this should be mpdp-display-grid

Reply:
agreed, this is fixed